app/src/main/java/org/p2p/wallet/striga/StrigaModule.kt

```diff
@@ -0,0 +1,19 @@
+package org.p2p.wallet.striga
+
+import org.koin.core.module.dsl.factoryOf
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.bind
+import org.koin.dsl.module
+import org.p2p.wallet.striga.onboarding.StrigaOnboardingContract
+import org.p2p.wallet.striga.onboarding.StrigaOnboardingPresenter
+import org.p2p.wallet.striga.repository.StrigaPresetDataLocalRepository
+import org.p2p.wallet.striga.repository.StrigaPresetDataInMemoryRepository
+
+object StrigaModule {
```

Comment:
Created separate StrigaSignupModule

Reply:
let's work in a separate `striga` folder as you said. 
It will have a single module for that for now. 

We'll see if it complex or not in the future